public/index.html

```diff
@@ -4,6 +4,14 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <!-- basic setting -->
+    <meta property="og:title" content="Smart Speaker's Commands Speaker(SCSS)"/>
+    <meta property="og:type" content="website"/>
+    <meta property="og:url" content="https://eishis.github.io/smart-speakers-commands-speaker/"/>
+    <meta property="og:image" content="https://eishis.github.io/smart-speakers-commands-speaker/public/img/command_ogp.jpg"/>
```

Comment:
you don't need use public directory in url. public directory can see root url, so you just set "https://eishis.github.io/smart-speakers-commands-speaker/img/command_ogp.jpg"